pages/algorithms/exposed-to-assistive-technologies.md

```diff
@@ -1,5 +1,8 @@
 ---
 title: Exposed to assistive technologies
+key: exposed-to-assistive-technologies
 ---
 
-Elements should be contained in the accessibility tree as described in [Core Accessibility API Mappings 1.1](https://www.w3.org/TR/core-aam-1.1/#mapping_general)
+Elements contained in the accessibility tree as described in [Core Accessibility API Mappings 1.1](https://www.w3.org/TR/core-aam-1.1/#mapping_general) and associated native markup language API mappings, e.g. [HTML Accessibility API Mappings](https://www.w3.org/TR/html-aam) and [SVG Accessibility API Mappings](https://www.w3.org/TR/svg-aam).
```

Comment:
Should this not be "or" instead of "and"?

Apart from the above question and Wilco his comment I approve of this definition.